modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -1,11 +1,28 @@
 // Module included in the following assemblies:
 // * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+// * post_installation_configuration/configuring-aws-outposts.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-aws-load-balancer-with-outposts_{context}"]
-= AWS Load Balancer Operator and AWS Outposts
+= Using the AWS Load Balancer Operator with AWS Outposts
 
-AWS Load Balancer Operator can work with AWS Outposts. Application Load Balancer is supported. Network Load balancer is not, so AWS Load Balancer Operator does not provision it on AWS Outposts subnets. Ingress resources must be annotated with the either outposts or regular subnets (but not both).
+You can configure the AWS Load Balancer Operator to provision an AWS Application Load Balancer in an Outpost.
+AWS Outposts does not support AWS Network Load Balancers.
+As a result, the AWS Load Balancer Operator cannot provision Network Load Balancers in an Outpost.
+
+You can create an individual AWS Application Load Balancer either in the cloud subnet or in the Outpost subnet, but they must be separate instances.
```

Comment:
I'm not sure I understand the meaning of "individual" and "must be separate instances" in this context. May be make this phrase simpler:
```suggestion
You can create an AWS Application Load Balancer either in the cloud subnet or in the Outpost subnet.
```